packages/web/components/complex/all-pools-table.tsx

```diff
@@ -463,14 +465,12 @@ export const AllPoolsTable: FunctionComponent<{
                           {pool.apr.toString()}
                         </p>
                       </Tooltip>
-                    ) : Boolean(pool.concentratedPoolDetail) ? (
-                      <ClAprBreakdownCell
-                        poolId={pool.queryPool.id}
-                        apr={pool.apr}
-                      />
-                    ) : (
+                    ) : Boolean(pool.concentratedPoolDetail) &&
+                      flags.aprBreakdown ? (
+                      <ClAprBreakdownCell poolId={pool.queryPool.id} />
+                    ) : flags._isInitialized ? (
                       pool.apr.toString()
-                    )
+                    ) : null
```

Comment:
I think this conditional is a little difficult to understand because it's too nested with ternaries. What are your thoughts on refactoring it like this: 

```tsx
let value: ReactNode | null;
if (isAPRTooHigh) {
  // Only display warning when APR is too high
  value = (
    <Tooltip
      className="w-5"
      content={t("highPoolInflationWarning")}
    >
      <p className="flex items-center gap-1.5">
        <Icon
          id="alert-triangle"
          className="h-4 w-4 text-osmoverse-400"
        />
        {pool.apr.toString()}
      </p>
    </Tooltip>
  );
} else if (Boolean(pool.concentratedPoolDetail) && flags.aprBreakdown) {
  value = <ClAprBreakdownCell poolId={pool.queryPool.id} />;
} else if (flags._isInitialized) {
  value = pool.apr.toString();
} else {
  value = null;
}

 <MetricLoaderCell
                  isLoading={
                    queriesOsmosis.queryIncentivizedPools.isAprFetching
                  }
                  value={value} />
```